proto/route.proto

```diff
@@ -6,44 +6,42 @@ import "types.proto";
 import "route_type.proto";
 
 message VnetDirect {
-    // destination vnet name if action_type is {vnet, vnet_direct}, a vnet other than eni's vnet means vnet peering
+    // destination vnet name if routing_type is {vnet, vnet_direct}, a vnet other than eni's vnet means vnet peering
     string vnet = 1;
     // overly_ip to lookup if routing_type is {vnet_direct}, use dst ip from packet if not specified
     optional types.IpAddress overlay_ip = 2;
 }
 
 message ServiceTunnel {
     // overlay ipv6 src ip if routing_type is {servicetunnel}
-    types.IpAddress overlay_sip = 1;
+    types.IpPrefix overlay_sip_prefix = 1;
     // overlay ipv6 dst ip if routing_type is {servicetunnel}
-    types.IpAddress overlay_dip = 2;
+    types.IpPrefix overlay_dip_prefix = 2;
     // underlay ipv4 src ip if routing_type is {servicetunnel}, this is the ST GW VIP (for ST traffic) or custom VIP
     types.IpAddress underlay_sip = 3;
     // underlay ipv4 dst ip to override if routing_type is {servicetunnel}, use dst ip from packet if not specified
     types.IpAddress underlay_dip = 4;
 }
 
 message Route {
-    route_type.RoutingType action_type = 1;
+    route_type.RoutingType routing_type = 1;
     oneof Action {
-        // destination vnet name if action_type is vnet,, a vnet other than eni's vnet means vnet peering
+        // destination vnet name if routing_type is vnet,, a vnet other than eni's vnet means vnet peering
         string vnet = 2;
-        // destination vnet name if action_type is vnet_direct,, a vnet other than eni's vnet means vnet peering
+        // destination vnet name if routing_type is vnet_direct,, a vnet other than eni's vnet means vnet peering
         route.VnetDirect vnet_direct = 3;
-        // appliance id if action_type is {appliance} 
+        // appliance id if routing_type is {appliance} 
         string appliance = 4;
-        // service tunnel if action_type is {service_tunnel}
+        // service tunnel if routing_type is {service_tunnel}
         route.ServiceTunnel service_tunnel = 5;
     }
-    // Metering policy lookup enable (optional), default = false
-    optional bool metering_policy_en = 6;
-    // Metering class-id, used if metering policy lookup is not enabled
-    optional uint64 metering_class = 7;
+    optional uint32 metering_class_or = 6;
```

Comment:
will it be better to use 8 here to avoid metering_policy_en and metering_class 7?

Reply:
fixed, deprecated old metering fields